### PR TITLE
added stuck_detector to bringup

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

**Review by floweisshardt — APPROVED** [review]

**Review by fmessmer — COMMENTED** [review]
do we also need this for the other robots?

File: cob_bringup/robots/cob4-2.xml [hunk 1]
```diff
@@ -35,6 +35,10 @@
 				<arg name="robot" value="$(arg robot)"/>
 				<arg name="pc" value="$(arg cob4-2-b1)"/>
 			</include>
+                        <include file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
```
Comment (NadiaHG) [inline cob_bringup/robots/cob4-2.xml:38]: could be the stuck_detector.launch file moved/copied to cob_bringup/drivers? For the consistency of cob_bringup..

**Comment by floweisshardt** [comment]
@ipa-fxm: this is rather one of the workarounds for https://github.com/ipa320/msh/issues/594. If this is solved properly, we can undo that again as it just adds another node and produces topic trafic. So it's ok to only have it in the 24/7 operated robots

**Review by fmessmer — CHANGES_REQUESTED** [review]
and I :+1: @ipa-nhg's request of moving the launch file to cob_bringup!

File: cob_bringup/robots/cob4-2.xml [hunk 2]
```diff
@@ -35,6 +35,10 @@
...
+                                <arg name="timeout" default="2.0"/>
+                                <arg name="threshold" default="0.174533"/>
+                        </include>
```
Comment (fmessmer) [inline cob_bringup/robots/cob4-2.xml:41]: tabs vs. spaces!

**Review by fmessmer — CHANGES_REQUESTED** [review]
and I :+1: @ipa-nhg's request of moving the launch file to cob_bringup!

File: cob_bringup/robots/cob4-2.xml (hunk 1 above)
Comment (benmaidel) [reply cob_bringup/robots/cob4-2.xml:38]: Copy :+1: Move :-1: 
If we move the launch file we will always need cob_robots (with thousands of deps) if we like to start the `stuck_detector`

**Comment by fmessmer** [comment]
> do we also need this for the other robots?

stuck detector is added to all "canopen base" robots in #622

File: cob_bringup/robots/cob4-2.xml (hunk 1 above)
Comment (NadiaHG) [reply cob_bringup/robots/cob4-2.xml:38]: copy is also ok for me!

**Comment by benmaidel** [comment]
@ipa-fxm no need to add it to all "canopen based" robots. Please see @ipa-fmw comment!

**Comment by fmessmer** [comment]
in #622, @ipa-mdl did a significant unification of launch and configuration files...
stuck detector is now part of the unified [`cob4_base.launch`](https://github.com/ipa320/cob_robots/pull/622/files#diff-d5e4351e5f3bc8f5fcb99475ed96d9e4R25)...thus, all robots using this file use the stuck detector..

also, as found yesterday, all robot platforms seem likely to show the base_freeze bug which is detected by the stuck detector...

**Comment by fmessmer** [comment]
In fact, this PR should not be merged...but rather #622

**Comment by mathias-luedtke** [comment]
Yes, don't merge this.

**Comment by mathias-luedtke** [comment]
Why?

**Comment by floweisshardt** [comment]
accidentally merged this while merging the enable/disable head branches. is reverted in https://github.com/ipa320/cob_robots/commit/12e46706c6ef4b9c112b9f49852b5f16ceb0bfee

**Pull request merged**